src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml

```diff
@@ -188,7 +189,7 @@
             var elementsToFilter = $("[data-filter-value]");
 
             // On ESC, clear the search box and display all features
-            if (e.keyCode == 27 || search == '') {
+            if (e.keyCode == EscapeKeyCode || search == '') {
```

Comment:
I think keyCode is deprecated. This is the correct way t check for Escape.

```suggestion
            if (e.key === 'Escape' || search == '') {
```

Reply:
Let me check, coz `keyCode` still working